src/lib/collectInventory.js

```diff
@@ -42,6 +42,13 @@ function collectKioskItems(kiosks, vendorDefs, _allItems, extraData) {
         return null;
       }
 
+      if (kioskEntry.flavorObjective) {
```

Comment:
Oh yeah, all emblems are in the kiosk so you only need to look here for these. Nice 🎉